consensus/state.go

```diff
@@ -1280,8 +1280,7 @@ func (cs *ConsensusState) finalizeCommit(height int64) {
 		panic(fmt.Sprintf("+2/3 committed an invalid block: %v", err))
 	}
 
-	cs.Logger.Info(fmt.Sprintf("Finalizing commit of block with %d txs", block.NumTxs),
-		"height", block.Height, "hash", block.Hash(), "root", block.AppHash)
+	cs.Logger.Info("height", block.Height, "hash", block.Hash(), "root", block.AppHash)
```

Comment:
the format of Info is `message, "arg1", arg1, ...`

```suggestion
	cs.Logger.Info("Finalizing commit of block", "height", block.Height, "hash", block.Hash(), "root", block.AppHash)